ruleset_test.inc

```diff
@@ -92,10 +92,10 @@ $my_theme_options = get_option( 'my_theme', false );
 if ( array_key_exists( 'key', $my_theme_options ) ) { // Error.
 }
 
-// Generic.PHP>NoSilencedErrors + WordPressVIPMinimum.VIP.RestrictedFunctions.file_get_contents_file_get_contents
+// Generic.PHP>NoSilencedErrors + WordPressVIPMinimum.Functions.RestrictedFunctions.file_get_contents_file_get_contents
 @file_get_contents( $foo ); // Error + Warning.
 
-// WordPressVIPMinimum.VIP.RegexpCompare + WordPress.VIP.SlowDBQuery
+// WordPressVIPMinimum.Performance.RegexpCompare + WordPress.VIP.SlowDBQuery
```

Comment:
Can this `WordPress.VIP.SlowDBQuery` be updated for this release as well now? Along with any other `WordPress.VIP` references.